lochness/mindlamp/__init__.py

```diff
@@ -9,10 +9,64 @@
 from io import BytesIO
 from pathlib import Path
 from typing import Tuple, List
+import pytz
+import time
+from datetime import datetime, timedelta
+import base64
+import re
 
 logger = logging.getLogger(__name__)
 
 
+def get_days_to_pull(Lochness):
+    '''Reads mindlamp_days_to_pull from Lochness loaded from the config.yml
+
+    If this variable is missing in the config.yml, set it to pull previous
+    100 days of data from today, from the mindlamp server. '''
+
+    value = Lochness.get('mindlamp_days_to_pull', dict())
+    if not isinstance(value, int):
+        return 100
+    return value
+
+
+def get_audio_out_from_content(activity_dicts, audio_file_name):
+    '''Separate out audio data from the content pulled from mindlamp API
+
+    Key Arguments:
+        activity_dicts: list of activity dictionaries, list.
+        audio_file_name: name of the audio file to be saved, str.
+
+    Returns:
+        activity_dicts_wo_sound: list of activity dictionaries without the
+                                 sound data. (sound data replaced to
+                                 'SOUND_{num}')
+
+    Notes:
+        `num` variables used in the function to capture all audio data, when 
+        there is more than one recording each day.
+    '''
+    activity_dicts_wo_sound = []
+
+    num = 0
+    for activity_events_dicts in activity_dicts:
+        if 'url' in activity_events_dicts['static_data']:
+            audio = activity_events_dicts['static_data']['url']
+            activity_events_dicts['static_data']['url'] = 'SOUND_{num}'
```

Comment:
The value appears to be a mistake. It should be `f'SOUND_{num}'`

Reply:
Thanks for this. I'll update this.

Reply:
aca77b9